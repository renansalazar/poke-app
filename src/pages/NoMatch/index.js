export default function NoMatch () {
  return <h2>PÃ¡gina no encontrada</h2>
}