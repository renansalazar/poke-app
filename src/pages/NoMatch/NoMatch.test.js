import React from 'react';
import {render} from '@testing-library/react';
import  NoMatch from '.';

describe('NoMatch component', () => {
  test('not found snapshot', () => {
    const { container } = render(<NoMatch />)
    expect(container.firstChild).toMatchSnapshot()
  });
});