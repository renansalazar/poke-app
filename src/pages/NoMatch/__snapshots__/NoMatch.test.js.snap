// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch component not found snapshot 1`] = `
<h2>
  PÃ¡gina no encontrada
</h2>
`;
